app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/data/database/model/Utils.kt

```diff
@@ -0,0 +1,37 @@
+package com.aptoide.android.aptoidegames.gamegenie.data.database.model
+
+import com.aptoide.android.aptoidegames.gamegenie.domain.ChatInteraction
+import com.aptoide.android.aptoidegames.gamegenie.domain.GameContext
+import com.aptoide.android.aptoidegames.gamegenie.domain.GameGenieChat
+import com.google.common.reflect.TypeToken
+import com.google.gson.Gson
+
+fun GameGenieHistoryEntity.toDomain(): GameGenieChat {
```

Comment:
i know these will only be used on the next ticket where we show the history, but it should have only been implemented there